src/org/omegat/util/WikiGet.java

```diff
@@ -421,11 +421,15 @@ private static String getStringContent(HttpURLConnection conn, String defaultCha
     public static class ResponseError extends IOException {
         public final int code;
         public final String message;
+        public final String body;
 
         public ResponseError(HttpURLConnection conn) throws IOException {
             super(conn.getResponseCode() + ": " + conn.getResponseMessage());
             code = conn.getResponseCode();
             message = conn.getResponseMessage();
+            try (InputStream in = conn.getErrorStream()) {
+                body = IOUtils.toString(in, "utf-8");
```

Comment:
It's a good practice on JDK 7+ to use `StandardCharsets.UTF-8.name()` rather than hardcoded "utf-8". 
https://stackoverflow.com/questions/6698354/where-to-get-utf-8-string-literal-in-java